estate/models/estate_property.py

```diff
@@ -0,0 +1,24 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+class EstateProperty(models.Model):
+	_name = "estate.property"
+	_description = "Real estate propert details model"
+
+	name = fields.Char(string = "Property Name", required=True)
```

Comment:
```suggestion
	name = fields.Char(string="Property Name", required=True)
```
don't add unnecessary spaces. remove them from all fields